documentation/homework/week1/src/components/todoList.js

```diff
@@ -0,0 +1,28 @@
+import React,{Component} from 'react';
+import TodoListItem from './TodoListItem';
+import TodoAdd from './todoAdd';
+
+class TodoList extends Component {
+
+    render() {
+        const todoList = this.props.listTodo;
+        const todoListItem = todoList.map((data,index) =>
+             <TodoListItem 
+                id={data.id} 
+                description={data.description} 
+                deadline={data.deadline} 
+                done={data.done} 
+                handleDoneClick={this.props.handleDoneClick}
+                removeTodo = {this.props.removeTodo}
```

Comment:
It's not a good practice to put space between prop name or prop value and equal like `removeTodo = {this.props.removeTodo}` instead `removeTodo={this.props.removeTodo}`